### PR TITLE
Add `inverse` prop to `Text` component

### PR DESCRIPTION
**Pull request by goliney**
Readme says:
> Ink uses chalk under the hood, so all its functionality is supported.

However, `inverse` functionality seems to be left behind.

image

### CODE REVIEW

**Comment by goliney** [comment]
@vadimdemedes please take a look

File: readme.md [hunk 1]
```diff
@@ -255,6 +256,21 @@ Dim the color (emit a small amount of light).
...
+
+Inverse background and foreground colors.
+
+```jsx
+<Text color="yellow" inverse>
+	Inversed Yellow
+</Text>
+```
+
+<img src="media/text-inverse.png" width="138">
```
Comment (vadimdemedes) [inline readme.md:272]: Can you use a jpg format instead?

Comment (goliney) [reply readme.md]: Done in https://github.com/vadimdemedes/ink/pull/347/commits/354d09a4317dbb61fdde1187b70d99c1997f650f

**Comment by vadimdemedes** [comment]
Thanks!

**Comment by vadimdemedes** [comment]
Released in [3.0.3](https://github.com/vadimdemedes/ink/releases/tag/v3.0.3).

**Pull request merged**